src/language-html/ast.js

```diff
@@ -16,6 +16,8 @@ class Node {
     for (const [key, value] of Object.entries(props)) {
       if (key in NODES_KEYS) {
         this._setNodes(key, value);
+      } else if (key === "parent") {
+        setNonEnumerableProperty(this, "parent", value);
```

Comment:
I think this will break `Node#clone()`, but it seems already broken before since properties like `.prev` is also non-enumerable

Reply:
Hmm, I just want to output the html AST with `--debug-print-ast` for other work, so what do you think about merging once the first implementation that specify the replacer function in the `JSON.stringify`?  I think other issues can be addressed later.